### PR TITLE
Gmail login

### PR DESCRIPTION
**Pull request by adityab17190**
Added following Changes

1) New Admin Login HTML Page with gmail sign-in button
2) Added route to admin login page in config./index.js
3) Added a route /api/adminLogin to check whether email id exists in admin table
4) Configured the admin login route in MainApp/index.js

### CODE REVIEW

**Comment by super9user** [comment]
+1

File: MainApp/public/app/views/google-Login.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  <div id="my-signin2"></div>
+
+  
+  <script>
+    function onSuccess(googleUser) {
+      console.log('Logged in as: ' + googleUser.getBasicProfile().getName());
+      console.log('User Email:'+googleUser.getBasicProfile().getEmail());
+      var email=googleUser.getBasicProfile().getEmail();
+      var request= new XMLHttpRequest();
+      var url="http://localhost:8080/api/adminLogin";
```
Comment (super9user) [inline MainApp/public/app/views/google-Login.html:58]: @adityab17190 This won't work well in production

**Comment by adityab17190** [comment]
@super9user Are you talking about the hardcoded url?

File: MainApp/public/app/views/google-Login.html (hunk 1 above)
Comment (adityab17190) [reply MainApp/public/app/views/google-Login.html:58]: If you are talking about the hardcoded URL http://localhost:8080/api/adminLogin,  my understanding was that everytime you call http://localhost:8080/login there is a sendFile() function called to send the html file. So changing the URL in the html file on the server would be enough.  But yeah, I will try to remove the url from the function and construct is using config variables. 
Thanks.

**Pull request merged**